tests/acceptance/features/coreApiWebdavOperations/refuseAccess.feature

```diff
@@ -27,7 +27,7 @@ Feature: refuse access
     Given using <dav-path-version> DAV path
     And user "Alice" has been created with default attributes and without skeleton files
     And user "Alice" has uploaded file "filesForUpload/textfile.txt" to "textfile0.txt"
-    And user "Alice" has been disabled
```

Comment:
This step is also used in other scenarios and the step is failing and might be is in the expected to failure file somewhere. So shall we replace this step with new step `And the user "Admin" has disabled user "Alice" using the Graph API` and coreect the other scenarios as well ?
CC @phil-davis @saw-jan @SwikritiT

Reply:
Note we also have these issue for this https://github.com/owncloud/ocis/issues/6729 https://github.com/owncloud/ocis/issues/1420

I think we should refactor them all but maybe in separate PR's ?